_includes/docs/0.4.x/gettingstarted.html

```diff
@@ -30,6 +30,11 @@
       You're going to use the <a target="_blank" href="http://gruntjs.com/">Grunt Task Runner</a> to automate your development process, in order to install it make sure you've installed Node.js and npm, then install grunt globally using npm:
       <pre>$ npm install -g grunt-cli</pre>
     </dd>
+    <dt>Gulp (Optional)</dt>
+    <dd>
+      You may use the <a target="_blank" href="http://gulpjs.com/">Gulp Task Runner</a> instead of Grunt.  You may use Gulp for Live Reload, Linting, and SASS or LESS compilation. Node must be installed first.
```

Comment:
@codydaig I think it's pretty obvious that you need npm.  Should this be removed for both Grunt and Gulp?